customLabels.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"encoding/json"
+	go_cache "github.com/patrickmn/go-cache"
+	"strings"
+	"time"
+	plog "github.com/prometheus/common/log"
+)
+
+// Custom labels for ConsumerGroups
+type CustomCGLagLabels struct {
+	labelByPrefix			map[string]string
+	labelCache			*go_cache.Cache
+}
+
+func NewCustomCGLagLabels(config string) (*CustomCGLagLabels, error){
+	labelByOwner := make(map[string][]string)
+	err := json.Unmarshal([]byte(config), &labelByOwner)
+	if err != nil {
+		return nil, err
+	}
+
+	labelByPrefix := make(map[string]string)
+	for owner, startWith := range  labelByOwner{
+		for _, startWith := range startWith {
+			labelByPrefix[startWith] = owner
+		}
+	}
+	labelCache := go_cache.New(1440*time.Minute, 60*time.Minute)
```

Comment:
what do you think of making this configurable by adding the two properties to the constructor and to the main itself?

Reply:
I agree